opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/__init__.py

```diff
@@ -369,16 +370,16 @@ def __init__(
         self._shutdown_once = Once()
         self._shutdown = False
 
-    def force_flush(self) -> bool:
+    def force_flush(self, timeout_millis: float = 10_000) -> bool:
 
         # FIXME implement a timeout
 
         for metric_reader in self._sdk_config.metric_readers:
-            metric_reader.collect()
+            metric_reader.collect(timeout_millis=timeout_millis)
```

Comment:
Should this be similar to below code where `timeout_millis` is total deadline instead of passing down same?

Reply:
Yup, thanks!